pairing/bn254/group.go

```diff
@@ -0,0 +1,80 @@
+package bn254
+
+import (
+	"crypto/cipher"
+
+	"github.com/drand/kyber"
+	"github.com/drand/kyber/group/mod"
+)
+
+type groupG1 struct {
+	common
+	*commonSuite
+	dst []byte
+}
+
+func (g *groupG1) String() string {
+	return "bn254.G1"
```

Comment:
This is meant to satisfy the [Stringer](https://pkg.go.dev/fmt#Stringer) interface, could you add the hex-encoded marshalled point in the output?

e.g.:
https://github.com/drand/kyber-bls12381/blob/29052f59fca6ddd71855ac18c1175162da69cba8/kyber_g1.go#L156

Same for the other groups.

Reply:
Implemented for all groups in bn254 👍